rules-tests/TypeDeclaration/Rector/FunctionLike/AddClosureParamTypeFromArgClassStringRector/Fixture/fixture.php.inc

```diff
@@ -15,10 +14,9 @@ SomeClass::someCall(SomeClass::class, function ($object) {
 
 namespace Rector\Tests\TypeDeclaration\Rector\FunctionLike\AddClosureParamTypeFromArgClassStringRector\Fixture;
 
-use SomeNamespace\SomeClass;
-use SomeNamespace\AnotherClass;
+use Rector\Tests\TypeDeclaration\Rector\FunctionLike\AddClosureParamTypeFromArgClassStringRector\Source\SimpleContainer;
 
-SomeClass::someCall(SomeClass::class, function (\SomeNamespace\SomeClass $object) {
+SimpleContainer::someCall(\Rector\Tests\TypeDeclaration\Rector\FunctionLike\AddClosureParamTypeFromArgClassStringRector\Source\SomeType::class, function (\Rector\Tests\TypeDeclaration\Rector\FunctionLike\AddClosureParamTypeFromArgClassStringRector\Source\SomeType $object) {
```

Comment:
By defalut, non-existing class are seen as strings. By both PHPStan and Rector.

That's why it's better to use existing class in tests, so we know it's an object and not a dummy string.